### DISCUSSION TITLE
FOR POSTING AN ERROR PLEASE READ THIS

### DISCUSSION THREAD

**Discussion by shivamguys** [shivamguys/irctc-cypress-automation#28] [Announcements]
### You can just run the command `npx cypress run --record --key 967b25a7-e4dd-4b5b-a143-d799131b2d3b --headed` after you have encountered any error this would record your run  and can be seen here 

https://cloud.cypress.io/projects/wy3npb/runs

This would help to have a look at your test run and would help to better resolve the issues.

**Comment by shivamguys** [comment]
`npx cypress run --record --key 967b25a7-e4dd-4b5b-a143-d799131b2d3b --headed`

**Reply by aman71711** [reply]
anyway or cmd i can schedule cypress to start operating on particular time like for example script automatically start working at 9:59:50 AM
